src/screens/Learn/Faq/Faq.style.tsx

```diff
@@ -1,33 +1,10 @@
 import styled from 'styled-components';
 import { mobileBreakpoint } from 'assets/theme/sizes';
-import theme from 'assets/theme';
-
-export const PageContainer = styled.div`
-  display: flex;
-  max-width: ${mobileBreakpoint};
-  margin: ${theme.spacing(3)}px auto ${theme.spacing(4)}px auto;
-`;
-
-export const PageContent = styled.main`
-  flex: 1 1 0;
-  padding: 1rem;
-`;
 
 export const Sidebar = styled.div`
```

Comment:
nearly all of the styles in Learn are shared and can live in `Learn.style.tsx`. we can probably nix this file altogether and move the sidebar styles somewhere shared?